sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -191,171 +185,45 @@ export interface AttestTpmOptions extends AttestationClientOperationOptions {
-    //
-    // (undocumented)
-    policyCertificates: PolicyCertificates_2;
-    // Warning: (ae-forgotten-export) The symbol "SigningCertificates" needs to be exported by the entry point index.d.ts
-    //
-    // (undocumented)
-    signingCertificates: SigningCertificates;
-}
-
-// @public (undocumented)
-export class GeneratedClientContext extends coreClient.ServiceClient {
-    constructor(credentials: coreAuth.TokenCredential, instanceUrl: string, options?: GeneratedClientOptionalParams);
-    // (undocumented)
-    apiVersion: string;
-    // (undocumented)
-    instanceUrl: string;
-}
-
-// @public (undocumented)
-export interface JsonWebKey {
-    alg?: string;
-    crv?: string;
-    d?: string;
-    dp?: string;
-    dq?: string;
-    e?: string;
-    k?: string;
-    kid?: string;
-    kty: string;
-    n?: string;
-    p?: string;
-    q?: string;
-    qi?: string;
-    use?: string;
-    x?: string;
-    x5C?: string[];
-    y?: string;
-}
-
-// @public (undocumented)
-export interface JsonWebKeySet {
-    keys?: JsonWebKey[];
-}
-
-// @public
-export enum KnownAttestationType {
+export const enum KnownAttestationType {
     OpenEnclave = "OpenEnclave",
     SgxEnclave = "SgxEnclave",
     Tpm = "Tpm"
 }
 
 // @public
-export enum KnownCertificateModification {
+export const enum KnownCertificateModification {
     IsAbsent = "IsAbsent",
     IsPresent = "IsPresent"
 }
 
 // @public
-export enum KnownDataType {
-    Binary = "Binary",
-    Json = "JSON"
-}
-
-// @public
-export enum KnownPolicyModification {
+export const enum KnownPolicyModification {
     Removed = "Removed",
     Updated = "Updated"
 }
 
-// @public
-export class PolicyCertificates {
-    constructor(client: AttestationClient);
-    add(policyCertificateToAdd: string, options?: coreClient.OperationOptions): Promise<PolicyCertificatesAddResponse>;
-    get(options?: coreClient.OperationOptions): Promise<PolicyCertificatesGetResponse>;
-    remove(policyCertificateToRemove: string, options?: coreClient.OperationOptions): Promise<PolicyCertificatesRemoveResponse>;
-}
-
-// @public
-export type PolicyCertificatesAddResponse = PolicyCertificatesModifyResponse;
-
-// @public
-export type PolicyCertificatesGetResponse = PolicyCertificatesResponse;
-
 // @public
 export interface PolicyCertificatesModificationResult {
     certificateResolution?: CertificateModification;
     certificateThumbprint?: string;
 }
 
-// @public
-export interface PolicyCertificatesModifyResponse {
-    token?: string;
-}
-
-// @public
-export type PolicyCertificatesRemoveResponse = PolicyCertificatesModifyResponse;
-
-// @public
-export interface PolicyCertificatesResponse {
-    token?: string;
-}
-
-// @public
-export interface PolicyCertificatesResult {
-    policyCertificates?: JsonWebKeySet;
-}
-
 // @public
 export type PolicyModification = string;
 
 // @public
 export class PolicyResult {
```

Comment:
can we replace this with an interface instead?

Reply:
Replace which? PolicyCertificatesResult is already an interface. However it should't be exported at all because JsonWebKeySet isn't exported ever.

Let me see what's going on here.

Reply:
The `AttestationSigner` class can be replaced with an interface without too much fuss, if that is what you intended.